runtime/doc/options.txt

```diff
@@ -4486,6 +4486,16 @@ A jump table for the options with a short description can be found at |Q_op|.
 	Insert mode completion.  When zero as much space as available is used.
 	|ins-completion-menu|.
 
+						*'pumblend'* *'pb'*
+'pumblend' 'pb'		number	(default 0)
+			global
+	Enables semi-transparency for the completion popupmenu. Valid values
+	are in the range from 0 for fully opaque popupmenu (disabled) to 100
+	for fully transparent background. Lower values 0-30 are typically most
+	useful.
+
+	Only takes effect in TUI with 'termguicolors' enabled.
```

Comment:
Should it be included in `option_set` event? Then the help might be changed to:

    UI-dependent. Supported by TUI with 'termguicolors' enabled.

Reply:
makes sense. Changed.